__tests__/RegistrationScreen.tests.js

```diff
@@ -1,14 +1,9 @@
 import "react-native";
-import React from "react";
-import { mount, shallow } from "enzyme";
+import * as React from "react";
```

Comment:
specifically what are you importing here? Best to elaborate here rather than use star.